### ISSUE TITLE
Permission woes make it impossible to dev on android device itself

### ISSUE THREAD

**Issue by HarrievG** [love2d/love-android#280]
For any future readers.
There is no way on a unrooted devices to use 11.3 > without having a shitton of permission pain.
Just downgrade to 11.3 and you will be able to edit lua files on your device in the good old /sdcard/lovegame.

_The core problem here is that you are not able to copy files to or edit files due to app permissions in_ **/sdcard/Android/data/org.love2d.android/files/games/lovegame**
 on an unrooted device
 
 With ADB you are able to create and copy files to that location. even with unrooted adb. So editing on pc and then copying to device is still a viable option to dev
 
 TLDR
 Love2d 11.3 > forces the usage of /sdcard/Android/data/org.love2d.android/files/games/lovegame, which is a protected location.
 _**If you want to be able to edit lua files and run directly on your android device use 11.3.**_

_Originally posted by @HarrievG in https://github.com/love2d/love-android/issues/253#issuecomment-2438577778_

**Comment by HarrievG** [comment]
To add:
IMHO this a major drawback of the current permission system. 
But the love2d android build should be able to be build _with_ the right permissions and an option to read from sdcard/lovegame as was possible before.

**Comment by MikuAuahDark** [comment]
Developing LOVE game **in** Android is not supported. Developing LOVE game **for** Android is.

Please see #194 as to the rationale behind all the permission changes. There's nothing we can do about it.
